Remove hidden tag for latest Fuse images #3193

apupier opened the pull request:
it allows to have them listed in catalog component types when using odo

Signed-off-by: Aurélien Pupier <apupier@redhat.com>

Fixes #3193


Steps to test the pull request

  1. minishift start
  2. install odo >= 0.19.0
  3. odo catalog component list
  4. the Fuse images should be listed

agajdosi commented:
Hi @apupier, thank you for the PR. I have a suggestion however which is whether you could also update the templates in "upstream" which for us is here: https://github.com/openshift/openshift-ansible/ in v3.10 and v3.11 branches. Respective locations:
- https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/openshift_examples/files/examples/x86_64/xpaas-streams/fis-image-streams.json
- https://github.com/openshift/openshift-ansible/blob/release-3.10/roles/openshift_examples/files/examples/v3.10/xpaas-streams/fis-image-streams.json

Keeping it in sync makes it much easier for us to maintain.

tsedmik commented:
FYI @apupier, @agajdosi maybe https://github.com/jboss-fuse/application-templates/blob/master/fis-image-streams.json is even more "upstream" than https://github.com/openshift/openshift-ansible/

apupier commented:
> Hi @apupier, thank you for the PR. I have a suggestion however which is whether you could also update the templates in "upstream" which for us is here: https://github.com/openshift/openshift-ansible/ in v3.10 and v3.11 branches. Respective locations:
> 
> * https://github.com/openshift/openshift-ansible/blob/release-3.11/roles/openshift_examples/files/examples/x86_64/xpaas-streams/fis-image-streams.json
> * https://github.com/openshift/openshift-ansible/blob/release-3.10/roles/openshift_examples/files/examples/v3.10/xpaas-streams/fis-image-streams.json
> 
> Keeping it in sync makes it much easier for us to maintain.

created issue upstream https://github.com/openshift/openshift-ansible/issues/11292
and provided 2 PRs:
- for 3.11 branch: https://github.com/openshift/openshift-ansible/pull/11293
- for 3.10 branch: https://github.com/openshift/openshift-ansible/pull/11294

apupier commented:
> Please update the files also in upstream. And change commit text to
> 
> ```
> Issue #3193 Remove hidden tag for latest Fuse images
> ```
> so it follows this repo's commit format.

commit message updated

agajdosi commented:
ok to test

agajdosi commented:
Hi @apupier, thank you for all the PRs. I can see that those 2 to openshift-ansible from which we take xpaas templates directly were merged. Please rebase this branch to latest master and once the tests passes we can merge.

Also please edit the commit message, you have lower case `I` in `issue` in commit message and upper case `I` is required. Sorry for formalism - no joy in that on my side really, but the check of the format is part PR requirements, so it is needed.

apupier commented:
> Hi @apupier, thank you for all the PRs. I can see that those 2 to openshift-ansible from which we take xpaas templates directly were merged. Please rebase this branch to latest master and once the tests passes we can merge.

rebased.
it has been accepted in openshift-ansible but not yet in upstream of upstream in Fuse Application templates repo but it is in good way. I think there will some more modifications to improve the ImageStream tag soon. I will provide another PR at that time :-)

> Also please edit the commit message, you have lower case `I` in `issue` in commit message and upper case `I` is required.

oups, updated

> Sorry for formalism - no joy in that on my side really, but the check of the format is part PR requirements, so it is needed.

no problem, i prefer when rules are enforced to help having a clean history. All the more that the CI is reporting the problem, I just didn't looked yet.

agajdosi commented:
@apupier Thank you for the patch!
